languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/removed.txt

```diff
@@ -366,8 +366,13 @@ fuissions	fuir	V ind impa 1 p
 fuissions	fuir	V sub pres 1 p
 fuissons	fuir	V imp pres 1 p
 fuissons	fuir	V ind pres 1 p
+<<<<<<<<< Temporary merge branch 1
 frustre	frustre	J e s
 frustre	frustre	N e s
 frustres	frustre	J e p
 frustres	frustre	N e p
+=========
 Aïd	Aïd	N f sp
+>>>>>>>>> Temporary merge branch 2
+franche	franche	J f s
+franches	franche	J f p
```

Comment:
This has to be fixed. Make a new clean pull-request. With so many commits, there are too many conflicts.

Reply:
Ok, new PR opened 
I close this one